### PR TITLE
core-services/image-mirroring: Mirror 4.12 image for COSA

### PR DESCRIPTION
**Pull request by travier**

### CODE REVIEW

File: core-services/image-mirroring-arm64/mapping_coreos_assembler_coreos_assembler [hunk 1]
```diff
@@ -1,4 +1,5 @@
 quay.io/coreos-assembler/coreos-assembler:latest registry.ci.openshift.org/coreos-arm64/coreos-assembler:latest
+quay.io/coreos-assembler/coreos-assembler:rhcos-4.11 registry.ci.openshift.org/coreos-arm64/coreos-assembler:4.12
```
Comment (mike-nguyen) [inline core-services/image-mirroring-arm64/mapping_coreos_assembler_coreos_assembler:2]:
```suggestion
quay.io/coreos-assembler/coreos-assembler:rhcos-4.12 registry.ci.openshift.org/coreos-arm64/coreos-assembler:4.12
```

File: core-services/image-mirroring/coreos/mapping_coreos_assembler_coreos_assembler [hunk 2]
```diff
@@ -1,4 +1,5 @@
 quay.io/coreos-assembler/coreos-assembler:latest registry.ci.openshift.org/coreos/coreos-assembler:latest
+quay.io/coreos-assembler/coreos-assembler:rhcos-4.11 registry.ci.openshift.org/coreos/coreos-assembler:4.12
```
Comment (mike-nguyen) [inline core-services/image-mirroring/coreos/mapping_coreos_assembler_coreos_assembler:2]:
```suggestion
quay.io/coreos-assembler/coreos-assembler:rhcos-4.12 registry.ci.openshift.org/coreos/coreos-assembler:4.12
```

**Comment by travier** [comment]
Thanks, updated

**Pull request merged**